diktat-rules/src/test/resources/test/paragraph3/trailing_comma/TrailingCommaExpected.kt

```diff
@@ -70,4 +79,45 @@ fun printMeanValue() {
         meanValue += year
     }
     println(meanValue/cars.size)
+}
+
+enum class SomeEnum(
+    val a: Int, val b: Int  ,// comment
```

Comment:
Does this rule run before spaces formatting? I.e., there won't be any extra spaces when the whole ruleset will be executed?

Reply:
Yes, the rules related to the indents and white spaces executes in last turn, while `TrailingCommaRule` runs about in the middle of execution